src/frontend/screens/WebView/index.tsx

```diff
@@ -32,27 +32,17 @@ export default function WebView() {
     lang = 'pt-BR'
   }
 
-  const epicLoginUrl = 'https://legendary.gl/epiclogin'
-
-  const epicStore = `https://www.epicgames.com/store/${lang}/`
-  const gogStore = `https://gog.com`
-  const wikiURL =
-    'https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/wiki'
-  const gogEmbedRegExp = new RegExp('https://embed.gog.com/on_login_success?')
-  const gogLoginUrl =
-    'https://auth.gog.com/auth?client_id=46899977096215655&redirect_uri=https%3A%2F%2Fembed.gog.com%2Fon_login_success%3Forigin%3Dclient&response_type=code&layout=galaxy'
-
   const trueAsStr = 'true' as unknown as boolean | undefined
   const { runner } = useParams() as { runner: Runner }
 
-  const urls = {
-    '/epicstore': epicStore,
-    '/gogstore': gogStore,
-    '/wiki': wikiURL,
-    '/loginEpic': epicLoginUrl,
-    '/loginGOG': gogLoginUrl,
-    '/loginweb/legendary': epicLoginUrl,
-    '/loginweb/gog': gogLoginUrl
+  const urls: { [index: string]: string } = {
+    '/epicstore': `https://www.epicgames.com/store/${lang}/`,
+    '/gogstore': `https://gog.com`,
+    '/wiki':
+      'https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/wiki',
+    '/loginweb/legendary': 'https://legendary.gl/epiclogin',
+    '/loginweb/gog':
+      'https://auth.gog.com/auth?client_id=46899977096215655&redirect_uri=https%3A%2F%2Fembed.gog.com%2Fon_login_success%3Forigin%3Dclient&response_type=code&layout=galaxy'
```

Comment:
Don't you think this made it more unreadable?
Another idea would be to move the list of URLs to a constants file and import them, making the file more cleaner.

Reply:
I mean, having the URLs as their own variables up top isn't very readable either IMO  
Having a separate file would probably be a good idea, I agree

Reply:
I've ended up not putting them in a separate file, see notes on the commit for details